.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: ci
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  delivery:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: Run Chef Delivery
+        uses: actionshub/chef-delivery@master
+        env:
+          CHEF_LICENSE: accept-no-persist
+
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: Run yaml Lint
+        uses: actionshub/yamllint@master
+
+  mdl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: Run Markdown Lint
+        uses: actionshub/markdownlint@master
+
+  rspec:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+      - name: Bundle
+        run: bundle install
+      - name: Run Rspec
+        run: rspec
```

Comment:
I don't think this is needed and should be covered by the delivery job above. This would also allow us to remove the ``Gemfile``.

Reply:
working on it.

Reply:
Gemfile is gone. Now using rspec in .delivery.  Thanks for pointing that out.

Reply:
Should be ready not.